services/src/main/java/io/druid/cli/PullDependencies.java

```diff
@@ -64,8 +64,7 @@
   private static final Logger log = new Logger(PullDependencies.class);
 
   private static final Set<String> exclusions = Sets.newHashSet(
-      "io.druid",
-      "com.metamx.druid",
```

Comment:
that would work but i think "io.druid" and "com.metamx.druid" are special and can be treated as such. and these are probably going to be present in all the extensions so it is probably safer to exclude them manually .
is there a downside to excluding them?